cmake/CompilerConfig.cmake

```diff
@@ -9,7 +9,7 @@ endif()
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(EXTRA_CXX_FLAGS "-Werror -Wall -Wpedantic -Wextra")
-set(CMAKE_CXX_FLAGS_RELEASE "-Ofast -flto -O3 -DRELEASE -DNDEBUG")
```

Comment:
The reason I made this change is because enabling link time optimisation globally was causing me issues (when compiling under GCC) when using static linking of detector modules. By using the CMake-way of enabling link time optimisation, we do make the code a bit more portable.

Reply:
Out of curiosity only - what kind if issues?

Reply:
It caused missing symbols.